<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mAcousto_pso</class>
 <widget class="QWidget" name="mAcousto_pso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>35</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>freq</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="mFreqCur">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string>kHz</string>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>6000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="mFreqSetVal">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="suffix">
        <string>kHz</string>
       </property>
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>6000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>BURST</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="mBurstVal">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="mBurstSetVal">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>power1 set</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="mPowerCurSet">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="mPowerSetVal">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>power2 set</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="mPowerCurSet_2">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="mPowerSetVal_2">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>gate fre1 set</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="mGateFreCurSet">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="mGateFreSetVal">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>gate fre2 set</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="mGateFreCurSet_2">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="mGateFreSetVal_2">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>mode</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="mRMode1">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="text">
        <string>模式1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mPowerModeSwt">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>50</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>in ctl</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="mRMode2">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="text">
        <string>模式2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mDebugSwt">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>50</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>off</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="mCorre">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="text">
        <string>corre</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>power mode</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="mPowerModeBtn">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="text">
        <string>内控      外控</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>debug mode1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="mDebunBtn">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="text">
        <string>开启     关闭</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>debug mode2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="mDebunBtn_2">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>55</height>
        </size>
       </property>
       <property name="text">
        <string>开启     关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
